src/lib.rs

```diff
@@ -49,6 +49,14 @@ pub fn tx_estimated_size_fjord(input: &[u8]) -> u64 {
         .max(MIN_TX_SIZE_SCALED)
 }
 
+/// Calculate the estimated compressed transaction size in bytes, scaled by 1e6.
```

Comment:
I think this is no longer scaled?

op geth refers to scaled as scaled up

https://github.com/ethereum-optimism/op-geth/blob/3e5073a7b5a40898521c0a4f35258acab8252a39/core/types/rollup_cost.go#L549-L551

which is currently equivalent to `tx_estimated_size_fjord`
so this function is the non scaled version if I understand correctly

Reply:
Yes, forgot to remove comment, sorry :)